src/modules/channelDetails/components/ChannelDetail.tsx

```diff
@@ -0,0 +1,276 @@
+                          margin="spacing-none spacing-none spacing-none spacing-xxxs"
+                        >
+                          {verifiedAliasChainIds.map((aliasChainId: number) => {
+                            const LogoComponent = LOGO_ALIAS_CHAIN[aliasChainId];
+                            return LogoComponent ? (
+                              <Box
+                                display="flex"
+                                css={css`
+                                  margin-left: -4px;
+                                `}
+                              >
+                                <LogoComponent
+                                  key={aliasChainId}
+                                  width={18}
+                                  height={18}
+                                />
+                              </Box>
+                            ) : null;
+                          })}
+                        </Box>
+                      )}
+                    </Box>
+                    <Box
+                      display="flex"
+                      gap="spacing-xxxs"
+                      alignItems="center"
+                    >
+                      <Text
+                        color="text-tertiary"
+                        variant="bs-regular"
+                      >
+                        {shortenText(channel?.channel || '', 10, 10) || ''}
+                      </Text>
+
+                      <CopyButton
+                        tooltipTitle="Copy Address"
+                        content={channel?.channel || ''}
+                      />
+                    </Box>
+                  </Box>
+                </Skeleton>
+                <Skeleton isLoading={isLoading}>
+                  <Box
+                    display="flex"
+                    gap="spacing-xs"
+                    css={css`
+                      flex-wrap: wrap;
+                    `}
+                    alignItems="center"
+                  >
+                    <Text
+                      color="text-tertiary-inverse"
+                      variant="c-regular"
+                    >
+                      {formatSubscriberCount(channel?.subscriber_count)} subscribers
+                    </Text>
+
+                    {tutotrialDetails && <ChannelTutorialContent tutotrialDetails={tutotrialDetails} />}
+                    {channel?.tags?.[0] && (
+                      <Tag
+                        label={channel.tags[0]}
+                        variant="info"
+                      />
+                    )}
+                  </Box>
+                </Skeleton>
+              </Box>
+            </Box>
+            <Box display={{ dp: 'inline', ml: 'none' }}>
+              <ChannelDetailSubscribe channel={channel} />
+            </Box>
+          </Box>
+        </Box>
+
+        <Skeleton isLoading={isLoading}>
+          <Box>
+            <Text
+              display={{ dp: 'inline', ml: 'none' }}
```

Comment:
```suggestion
              display={{ initial: 'inline', ml: 'none' }}
```

Fix this everywhere in this PR.